appengine/standard/endpoints-frameworks-v2/iata/app.yaml

```diff
@@ -0,0 +1,34 @@
+runtime: python27
+threadsafe: true
+api_version: 1
+basic_scaling:
+  max_instances: 2
+
+#[START_EXCLUDE]
```

Comment:
This shouldn't be necessary.

Reply:
These are in the app.yaml in appengine/standard/endpoints-frameworks-v2/echo; do you know why they're there? I just mimicked that.